src/tests/Microsoft.PowerFx.Connectors.Tests/OpenApiParserTests.cs

```diff
@@ -988,7 +989,50 @@ public async Task DataverseTest2(string swaggerFile)
         }
 
         [Fact]
-        public async Task PowerAppsCardsTest()
+        public async Task CardsForPowerApps_Invoke()
+        {
+            using var testConnector = new LoggingTestServer(@"Swagger\CardsForPowerApps.json");
+            using var httpClient = new HttpClient(testConnector);
+            using PowerPlatformConnectorClient client = new PowerPlatformConnectorClient("https://tip1002-002.azure-apihub.net", "7592282b-e371-e3f6-8e04-e8f23e64227c" /* environment Id */, "shared-cardsforpower-eafc4fa0-c560-4eba-a5b2-3e1ebc63193a" /* connectionId */, () => "eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsIng1dC...", httpClient) { SessionId = "a41bd03b-6c3c-4509-a844-e8c51b61f878" };
+
+            BaseRuntimeConnectorContext runtimeContext = new TestConnectorRuntimeContext("DV", client, console: _output);
+
+            ConnectorFunction[] functions = OpenApiParser.GetFunctions(new ConnectorSettings("DV") { Compatibility = ConnectorCompatibility.SwaggerCompatibility }, testConnector._apiDocument).ToArray();
+            ConnectorFunction createCardInstance = functions.First(f => f.Name == "CreateCardInstance");
+
+            testConnector.SetResponseFromFile(@"Responses\CardsForPowerApps_CreateCardInstance.json");
+            var result = await createCardInstance.InvokeAsync(
+                new FormulaValue[]
+                {
+                   FormulaValue.New("card"),
+                   FormulaValue.NewRecordFromFields(
+                           new NamedValue("inputs", FormulaValue.NewRecordFromFields(
+                                new NamedValue("property1", FormulaValue.New("test1")),
+                                new NamedValue("property2", FormulaValue.New("test2"))))),
+                },
+                runtimeContext,
+                CancellationToken.None).ConfigureAwait(false);
+
+            string input = testConnector._log.ToString();
+            Assert.Equal("AdaptiveCard", (((RecordValue)result).GetField("type") as StringValue).Value);
+            Assert.Equal(
+                @"POST https://tip1002-002.azure-apihub.net/invoke
+ authority: tip1002-002.azure-apihub.net
+ Authorization: Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsIng1dC...
+ path: /invoke
+ scheme: https
+ x-ms-client-environment-id: /providers/Microsoft.PowerApps/environments/7592282b-e371-e3f6-8e04-e8f23e64227c
+ x-ms-client-session-id: a41bd03b-6c3c-4509-a844-e8c51b61f878
+ x-ms-request-method: POST
+ x-ms-request-url: /apim/cardsforpowerapps/shared-cardsforpower-eafc4fa0-c560-4eba-a5b2-3e1ebc63193a/cards/cards/card/instances
+ x-ms-user-agent: PowerFx/1.3.0-local

```

Comment:
>PowerFx/1.3.0-local [](http://example.com/codeflow?start=18&length=19)

I see other test cases using a different pattern: `PowerFx/{version}`.
Any reason why using a literal version number here instead of the placeholder?

Reply:
Addressed